libraries/lib-wx-init/HelpSystem.cpp

```diff
@@ -47,6 +47,8 @@ const wxString HelpSystem::HelpHostname = wxT("manual.audacityteam.org");
 const wxString HelpSystem::HelpServerHomeDir = wxT("/");
 const wxString HelpSystem::HelpServerManDir = wxT("/man/");
 
+const wxString HelpSystem::AudacitySupportLink = wxT("gitbook.com/open/library/audacity/audacity");
```

Comment:
this here should be support.audacityteam.org - the link you used is gitbooks' overview site :)